foundationdb-kubernetes-sidecar/sidecar.py

```diff
@@ -359,7 +360,12 @@ def send_text(self, text, code=200, content_type="text/plain", add_newline=True)
         self.end_headers()
         self.wfile.write(response)
 
-    def check_request_cert(self):
+    def check_request_cert(self, path):
+        # The ready endpoint will not require a client certificate
+        # to allow the kubelet to do HTTP probes.
+        if path == "/ready":
```

Comment:
Since we require a certificate on the TLS layer we can drop that. In general it would be good to have a way to use the HTTP probe.